ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: openstack-installer
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.10"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.10"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: installer-rehearse
+  interval: 960h
+  steps:
+    cluster_profile: aws
```

Comment:
Just like what you mentioned in the meeting, sounds like the following workflow has dependency on `cluster_profile`, to help the team easy category workflow yaml files and easy to update placeholder workflow, maybe we can add several tests for each platform. something like:
```
tests:
- as: installer-rehearse-aws
  interval: 960h
  steps:
    cluster_profile: aws
    workflow: cucushift-installer-rehearse-aws
- as: installer-rehearse-gcp
  interval: 960h
  steps:
    cluster_profile: gcp
    workflow: cucushift-installer-rehearse-gcp
- as: installer-rehearse-azure
  interval: 960h
  steps:
    cluster_profile: azure
    workflow: cucushift-installer-rehearse-azure
- as: installer-rehearse-vsphere
  interval: 960h
  steps:
    cluster_profile: vsphere
    workflow: cucushift-installer-rehearse-vsphere
- as: installer-rehearse-packet
  interval: 960h
  steps:
    cluster_profile: packet
    workflow: cucushift-installer-rehearse-packet
``` 
Then user need to drop their install workflow into the proper folder. Then update the `workflow` value here to the creating workflow, no need to touch with other files.

Do you think that is reasonable?

Reply:
But whatever, the PR looks good to me, I can follow up to implement my proposal.  Thanks for this.